iphone/Maps/Classes/MapViewController.mm

```diff
@@ -521,6 +521,8 @@ - (void)processMyPositionStateModeEvent:(MWMMyPositionMode)mode {
 }
 
 - (void)processMyPositionPendingTimeout {
+  if (UIApplication.sharedApplication.applicationState != UIApplicationStateActive)
```

Comment:
Is ignoring the fired timer from the core the best way to fix the issue? What internal state will be there after the timer has fired?
For example, shouldn't the timer be stopped when the app goes into the background?